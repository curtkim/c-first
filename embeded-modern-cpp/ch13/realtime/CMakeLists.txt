cmake_minimum_required(VERSION 3.5.1)
project(realtime)
add_executable(realtime realtime.cpp)
target_link_libraries(realtime PUBLIC pthread)

