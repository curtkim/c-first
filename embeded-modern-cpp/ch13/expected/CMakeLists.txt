cmake_minimum_required(VERSION 3.5.1)
project(expected)
add_executable(expected expected.cpp)
target_link_libraries(expected PUBLIC pthread)
target_compile_features(expected PUBLIC cxx_std_17)
